### ISSUE TITLE
No elapsed time recorded in 0.85

### ISSUE THREAD

**Issue by AndyA** [houseabsolute/test-class-moose#81]
Running this test

```perl
# t/tests/TestFor/Sleep.pm
package TestFor::Sleep;
use Test::Class::Moose;

sub test_sleep {
    my $test = shift;
    sleep 5;
    is 1, 1, "Yes!";
}

1;
```

with this loader

```perl
# t/loader.t
use strict;
use warnings;

use Test::Class::Moose::Load 't/tests';
use Test::Class::Moose::Runner;
use Data::Dumper;

my $runner = Test::Class::Moose::Runner->new;

$runner->runtests;
my $rep = $runner->test_report;
print Dumper($rep);
```

gives this output with 0.85

```perl
$VAR1 = bless(
  { 'test_classes' => [
      bless(
        { 'start_time' => '1504601210.77945',
          '_end_benchmark' =>
           bless( ['1504601210.77943', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
          'notes'          => {},
          'name'           => 'TestFor::Sleep',
          'test_instances' => [
            bless(
              { '_start_benchmark' =>
                 bless( ['1504601210.77974', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                'test_shutdown_method' => bless(
                  { 'instance' => $VAR1->{'test_classes'}[0]{'test_instances'}[0],
                    '_end_benchmark' =>
                     bless( ['1504601215.78519', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                    'num_tests_run' => 0,
                    'start_time'    => '1504601215.7852',
                    '_start_benchmark' =>
                     bless( ['1504601215.7852', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                    'notes'    => {},
                    'name'     => 'test_shutdown',
                    'end_time' => '1504601215.78519',
                    'passed'   => 0
                  },
                  'Test::Class::Moose::Report::Method'
                ),
                'passed'              => 1,
                'end_time'            => '1504601210.77971',
                'name'                => 'TestFor::Sleep',
                'notes'               => {},
                'test_startup_method' => bless(
                  { 'notes'    => {},
                    'name'     => 'test_startup',
                    'end_time' => '1504601210.78053',
                    'passed'   => 0,
                    '_start_benchmark' =>
                     bless( ['1504601210.78056', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                    'num_tests_run' => 0,
                    'start_time'    => '1504601210.78056',
                    'instance'      => $VAR1->{'test_classes'}[0]{'test_instances'}[0],
                    '_end_benchmark' =>
                     bless( ['1504601210.78053', '0.28', '0.04', '0', '0', 0], 'Benchmark' )
                  },
                  'Test::Class::Moose::Report::Method'
                ),
                '_end_benchmark' =>
                 bless( ['1504601210.77971', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                'start_time'   => '1504601210.77974',
                'test_methods' => [
                  bless(
                    { 'start_time'           => '1504601210.78071',
                      'tests_planned'        => 1,
                      'num_tests_run'        => 1,
                      'test_teardown_method' => bless(
                        { 'passed'   => 0,
                          'end_time' => '1504601215.78501',
                          'name'     => 'test_teardown',
                          'notes'    => {},
                          '_start_benchmark' =>
                           bless( ['1504601215.78505', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                          'start_time'    => '1504601215.78505',
                          'num_tests_run' => 0,
                          '_end_benchmark' =>
                           bless( ['1504601215.78501', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                          'instance' => $VAR1->{'test_classes'}[0]{'test_instances'}[0]
                        },
                        'Test::Class::Moose::Report::Method'
                      ),
                      '_end_benchmark' =>
                       bless( ['1504601210.7807', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                      'instance'          => $VAR1->{'test_classes'}[0]{'test_instances'}[0],
                      'passed'            => 1,
                      'test_setup_method' => bless(
                        { 'num_tests_run' => 0,
                          'start_time'    => '1504601210.78074',
                          'instance'      => $VAR1->{'test_classes'}[0]{'test_instances'}[0],
                          '_end_benchmark' =>
                           bless( ['1504601210.78073', '0.28', '0.04', '0', '0', 0], 'Benchmark' ),
                          'notes'    => {},
                          'name'     => 'test_setup',
                          'end_time' => '1504601210.78073',
                          'passed'   => 0,
                          '_start_benchmark' =>
                           bless( ['1504601210.78074', '0.28', '0.04', '0', '0', 0], 'Benchmark' )
                        },
                        'Test::Class::Moose::Report::Method'
                      ),
                      'end_time' => '1504601210.7807',
                      'name'     => 'test_sleep',
                      'notes'    => {},
                      '_start_benchmark' =>
                       bless( ['1504601210.78071', '0.28', '0.04', '0', '0', 0], 'Benchmark' )
                    },
                    'Test::Class::Moose::Report::Method'
                  )
                ]
              },
              'Test::Class::Moose::Report::Instance'
            )
          ],
          'end_time' => '1504601210.77943',
          'passed'   => 1,
          '_start_benchmark' =>
           bless( ['1504601210.77945', '0.28', '0.04', '0', '0', 0], 'Benchmark' )
        },
        'Test::Class::Moose::Report::Class'
      )
    ],
    'end_time'    => '1504601210.77847',
    'start_time'  => '1504601210.77851',
    'is_parallel' => 0,
    '_end_benchmark' =>
     bless( ['1504601210.77847', '0.27', '0.04', '0', '0', 0], 'Benchmark' ),
    '_start_benchmark' =>
     bless( ['1504601210.77851', '0.27', '0.04', '0', '0', 0], 'Benchmark' )
  },
  'Test::Class::Moose::Report'
);
```

Although the test case takes 5 seconds to run, start_time and end_time are nearly identical:

```
'start_time' => '1504601210.78071',
'end_time' => '1504601210.7807',
```

Reverting to 0.84 makes the problem go away.

**Comment by autarch** [comment]
Thanks for the bug report! This will be fixed shortly.

**Comment by autarch** [comment]
Fixed in 0.86
